.github/actions/ci/action.yml

```
@@ -24,5 +27,8 @@ runs:
       shell: bash
     - run: make start-contract-test-service-bg
       shell: bash
-    - run: make run-contract-tests
-      shell: bash
+    - uses: launchdarkly/gh-actions/actions/contract-tests@contract-tests-v1.0.0
```

```suggestion
    - uses: launchdarkly/gh-actions/actions/contract-tests@contract-tests-v1.0.1
```

Having to remember to bump patches is going to be annoying. It's already happened with Matthew's repos too.